### DISCUSSION TITLE
Module to configure metadata quality

### DISCUSSION THREAD

**Discussion by IGNF-Xavier** [geonetwork/geonetwork-ui#930] [Ideas]
I suggest to support other metadata (services, reuse, etc) for the rendering of the metadata quality

**Comment by jahow** [comment]
Sorry, I'm not sure I understood this correctly: does that mean computing the metadata quality of other kinds of metadata?

**Comment by IGNF-Xavier** [comment]
Yes.
In other words, the widget metadata quality can be configured to verify attributesof other medata. (attributes eventually defined by the administrator)

**Reply by jahow** [reply]
This would be very valuable I think; Having a custom "formula" for computing the metadata quality could be done at two levels:
* in the default.toml config, by specifying fields & weights
* by allowing to inject a custom service for computing the md quality (e.g. in a custom app)

Technically this might be tricky because the md quality is also computed in the index through an ES pipeline.
